.github/workflows/ci.yml

```diff
@@ -132,7 +132,7 @@ jobs:
       - name: check esp32c2-hal (async, i2c)
         run: cd esp32c2-hal/ && cargo +nightly check --example=embassy_i2c --features=embassy,embassy-time-systick,async
       - name: check esp32c2-hal (interrupt-preemption)
-        run: cd esp32c2-hal/ && cargo check --example=interrupt_preemption --features=interrupt-preemption
+        run: cd esp32c2-hal/ && cargo +nightly check --example=interrupt_preemption --features=interrupt-preemption
```

Comment:
`nightly` is the only toolchain available for this job, do we really need to be explicit about it?

Reply:
Consistency with the other steps - everything else spells out `+nightly` here